pkg/cmd/quota/quota_list.go

```diff
@@ -64,9 +65,49 @@ func newCmdQuotaList(ctx api.Context) *cobra.Command {
 				return enc.Encode(quotas)
 			}
 
+			tableHeader := []string{"NAME", "CPU CONSUMED", "MEMORY CONSUMED", "DISK CONSUMED", "GPU CONSUMED"}
+			table := cli.NewTable(ctx.Out(), tableHeader)
+
 			for _, quota := range quotas {
-				fmt.Fprintln(ctx.Out(), quota.Role)
+				quotaInfo := map[string]string{"cpus": "", "mem": "", "disk": "", "gpus": ""}
+
+				for info := range quotaInfo {
+					if limit, ok := quota.Limit[info].(float64); ok {
+						var cons float64
+						if consumed, ok := quota.Consumed[info].(float64); ok {
+							cons = consumed
+						}
+						percent := fmt.Sprintf("%.2f%%", cons*100/limit)
+						var desc string
+						switch info {
+						case "cpus", "gpus":
+							desc = fmt.Sprintf("%.0f of %.0f Cores", cons, limit)
+						case "mem", "disk":
+
+							if limit < 1000 {
+								desc = fmt.Sprintf("%.0fMiB of %.0fMiB", cons, limit)
```

Comment:
do we want a space between the value an MiB? That's how it is in the UI.

Reply:
<img width="510" alt="Screenshot 2019-07-31 à 17 46 11" src="https://user-images.githubusercontent.com/2117580/62226845-20ad5f00-b3bb-11e9-9901-6f574d6a6939.png">
I used the Invision website but you're correct, the new UI shows spaces.